dev-util/intel-graphics-compiler/intel-graphics-compiler-1.0.9636.ebuild

```diff
@@ -0,0 +1,112 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_BUILD_TYPE="Release"
+LLVM_MAX_SLOT="13"
+MY_PN="igc"
+MY_P="${MY_PN}-${PV}"
+#PYTHON_COMPAT=( python3_{8..10} )
+PYTHON_COMPAT=( python3_10 )
+
+inherit cmake flag-o-matic llvm python-any-r1 git-r3
+
+DESCRIPTION="LLVM-based OpenCL compiler for OpenCL targetting Intel Gen graphics hardware"
+HOMEPAGE="https://github.com/intel/intel-graphics-compiler"
+SRC_URI="https://github.com/intel/${PN}/archive/${MY_P}.tar.gz -> ${P}.tar.gz"
+EGIT_REPO_URI="https://github.com/KhronosGroup/SPIRV-Tools.git"
+EGIT_REPO_URI1="https://github.com/KhronosGroup/SPIRV-Headers.git"
+EGIT_REPO_URI2="https://github.com/intel/vc-intrinsics.git"  EGIT_COMMIT2="8ee879314584e6630688b0a3b290d065dcabb383"
+# protobuf version 3.19.1 - but it is unnecesary 
+#EGIT_REPO_URI3="https://github.com/protocolbuffers/protobuf" EGIT_COMMIT3="7c40b2df1fdf6f414c1c18c789715a9c948a0725"
```

Comment:
Well, if not needed, than remove it?

Reply:
Only for documentation. In final realise it should be remove.